app/src/main/java/com/example/demomvp/data/model/Task.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.demomvp.data.model
+
+import android.content.ContentValues
+import android.database.Cursor
+
+data class Task(val mId: Int, val mTitle: String, val mDescription: String) {
```

Comment:
```suggestion
data class Task(val id: Int, val title: String, val description: String) {
```
Không dùng `convention` của java nhé